packages/blade/src/components/Typography/Text/Text.tsx

```diff
@@ -66,14 +69,17 @@ const getProps = <T extends { variant: TextVariant }>({
   };
 
   if (variant === 'body') {
-    props.fontSize = 100;
-    props.fontStyle = 'normal';
-    props.lineHeight = 'l';
+    if (size === 'small') {
+      props.fontSize = 75;
+    }
   } else if (variant === 'caption') {
     if (weight === 'bold') {
       throw new Error(`[Blade: Text]: weight cannot be 'bold' when variant is 'caption'`);
     }
-    props.fontSize = 25;
+    if (size === 'small') {
+      throw new Error(`[Blade: Text]: size cannot be 'small' when variant is 'caption'`);
+    }
+    props.fontSize = 50;
     props.fontStyle = 'italic';
     props.lineHeight = 's';
```

Comment:
Lineheight of `s` is good enough still since we increased the fontSize from 25 to 50?

Reply:
yep that was a typo on figma in the past we just rectfied it, @saurav12 to confirm if the lineheight is good enough even with `50`.

Reply:
Need to check it once. I will revert once I'm done with the changes on Figma. Have to check it visually.

Reply:
this is resolved